xarray/test/test_dataarray.py

```diff
@@ -1328,6 +1328,22 @@ def test_reduce(self):
         expected = DataArray(5, {'c': -999})
         self.assertDataArrayIdentical(expected, actual)
 
+    def test_quantile(self):
+        for method in ['linear', 'lower', 'higher', 'nearest', 'midpoint']:
```

Comment:
No need to test all these methods here, since we have no logic specific to them. It's enough to verify (once) that the `method` argument gets passed on to NumPy.